lib/use_cases/case_ready_for_automation.rb

```diff
@@ -0,0 +1,17 @@
+module UseCases
+  class CaseReadyForAutomation
+    def execute(patch_code:)
+      patch_codes_allowed_for_automation.include? patch_code
```

Comment:
Nit pick: I like brackets.

```suggestion
      patch_codes_allowed_for_automation.include?(patch_code)
```